db/src/schema.rs

```diff
@@ -250,3 +250,88 @@ impl SchemaTypeChecking for Schema {
         }
     }
 }
+
+
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use self::edn::NamespacedKeyword;
+
+    fn add_attribute(schema: &mut Schema, 
+            ident: NamespacedKeyword, 
+            entid: Entid, 
+            index: bool,
```

Comment:
Rather than all of these opaque positional arguments, could you just take the ident/entid and an `Attribute` instance? That allows callers to name the fields as they construct the attribute.